### PR TITLE
Submit application story 6

### PR DESCRIPTION
**Pull request by Sergio-Azcona**
thanks for pair programming and being a rubber duck!!!

### CODE REVIEW

File: spec/features/applications/new_spec.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@
         expect(page).to have_field(:city)
         expect(page).to have_field(:state)
         expect(page).to have_field(:zip_code)
-        expect(page).to have_field(:description)
+        # expect(page).to have_field(:description)
```
Comment (Tscasady) [inline spec/features/applications/new_spec.rb:60]: Should this expectation be commented out? The description and bid_reason are two separate things right?

File: spec/features/applications/show_spec.rb [hunk 2]
```diff
@@ -150,4 +151,82 @@
...
+
+        visit "/applications/#{@app3.id}"
+      
+        expect(@app3.pets.present?).to eq true
+        expect(@app3.status).to eq('In Progress')
+        expect(page).to have_field(:bid_reason)
+        expect(page).to have_button('Submit Application')
+      end
+
+      describe 'when user completes the worthy candidate and clicks the submit button' do
```
Comment (Tscasady) [inline spec/features/applications/show_spec.rb:180]: Consider rephrasing 'worthy candidate', could be more clear that this is referring to the bid_reason field.

File: spec/features/applications/show_spec.rb [hunk 3]
```diff
@@ -150,4 +151,82 @@
...
+
+          expect(current_path).to eq("/applications/#{@app3.id}")
+          
+          within("#show_data") do
+          expect(page).to_not have_content('In Progress')
+            expect(page).to have_content('Pending')  
+          end
+        end 
+
+        it "updates the application status 'Pending' once an application is submitted" do
```
Comment (Tscasady) [inline spec/features/applications/show_spec.rb:218]: Copied 'it' description from test above. Should read something like 'Add a Pet section is no longer visible when application is not In Progress.'

**Review by Tscasady — CHANGES_REQUESTED** [review]
Good work, all these tests are very thorough and clear and the implementation seems solid. Some small changes to be made to one of the descriptions of a test and then it is ready to merge.

File: spec/features/applications/new_spec.rb (hunk 1 above)
Comment (Sergio-Azcona) [reply spec/features/applications/new_spec.rb:60]: ah! good catch. I had commented it out mistakenly

**Comment by Sergio-Azcona** [comment]
i refactored this section and will push it uo. I think you can close this

**Comment by Sergio-Azcona** [comment]
actually, idk that i could! close complete

**Pull request closed without merging**